tests/skipped_tests.tbl

```diff
@@ -1141,6 +1141,8 @@ tests/third_party/cupy/random_tests/test_distributions.py::TestDistributionsDiri
 tests/third_party/cupy/random_tests/test_distributions.py::TestDistributionsExponential_param_0_{scale_shape=(), shape=(4, 3, 2)}::test_exponential
 tests/third_party/cupy/random_tests/test_distributions.py::TestDistributionsExponential_param_1_{scale_shape=(), shape=(3, 2)}::test_exponential
 tests/third_party/cupy/random_tests/test_distributions.py::TestDistributionsExponential_param_2_{scale_shape=(), shape=None}::test_exponential
+tests/third_party/cupy/random_tests/test_distributions.py::TestDistributionsGeometric_param_0_{p_shape=(), shape=(4, 3, 2)}::test_geometric
+tests/third_party/cupy/random_tests/test_distributions.py::TestDistributionsGeometric_param_1_{p_shape=(), shape=(3, 2)}::test_geometric
```

Comment:
Is this needed for checking you function logic?

Reply:
Function `multiply` is used in the tests and works correct there. To prove that I added separate test with similar check.

Reply:
if multiply is used in current tests, then it is ok.

Reply:
> if multiply is used in current tests, then it is ok.

non correct comment here. My fault.

Reply:
I thinks so.